Assets/Scripts/LevelManager.cs

```diff
@@ -72,6 +72,10 @@ public class LevelManager : MonoBehaviour {
     // Update is called once per frame
     void Update() {
         if (Input.GetKeyDown("f")) {
+            if(explosionManager.GetPlacedBombCount() == player.bombs)
+            {
+                return;
```

Comment:
Fajnie by było pokazać jakiś komunikat playerowi albo np. zrobić podświetlenie bomby hud'a bo tak to nie wiadomo dlaczego nie mogę postawić bomby;